back/appsync/serverless.yml

```diff
@@ -14,12 +14,15 @@ provider:
   environment:
     SES_AWS_REGION: ${self:provider.region}
     MAILBOX_ADDR: ${env:MAILBOX, 'ideas@beta.dailyidea.com'}
+    SENTRY_ENVIRONMENT: ${opt:stage, self:provider.stage}
+    SENTRY_DSN: ${env:SENTRY_DSN}
 
 plugins:
   - serverless-offline
   - serverless-appsync-plugin
   - serverless-python-requirements
   - serverless-prune-plugin
+  - serverless-sentry
```

Comment:
same issue here as with python.
serverless-sentry is depends on old serverless-sentry-lib which depends on old sentry pacakges.
there is a newer ( 2.0)  version of serverless-sentry-lib which work on top of latest sentry packages, but as i see serverless-sentry probably don't support it so it seems we will need to update function with wrappers than manually
https://github.com/arabold/serverless-sentry-lib#updating-your-code

Reply:
v. 2.0.0 was released just 10 hours ago, maybe it's too early to use it?

Reply:
Ok, sentry plugin got update to latest lib so we can just use its latest version now https://github.com/arabold/serverless-sentry-plugin

Reply:
2 days ago 
v2.0.0 …
2.0.0

Reply:
Ok, upgraded and tested